src/views/Home.vue

```diff
@@ -126,15 +136,46 @@ export default defineComponent({
   },
   setup() {
     const router = useRouter();
-
+    const ionCard = ref();
+    const scrollItem = ref();
     return {
       router,
       ticketOutline,
       shirtOutline,
       sendOutline,
       calendarOutline,
-      settingsOutline
+      settingsOutline,
+      chevronBackOutline,
+      chevronForwardOutline,
+      ionCard,
+      scrollItem
     }
+  },
+   methods: {
+    async nextItems(event: any) {
+        this.ionCard.scrollLeft += 350;
+        const ionCard = this.ionCard;
+        this.ionCard.style.overflow = 'hidden';
+        const animation = createAnimation()
+        .addElement(this.scrollItem)
+        .easing('linear')
+        .duration(1500)
```

Comment:
Are we sure about this duration?

Reply:
currently I have given this minimum of 1500